src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java

```diff
@@ -222,16 +219,7 @@ public void setLastStateModifiedDate(LocalDateTime lastStateModifiedDate) {
         this.lastStateModifiedDate = lastStateModifiedDate;
     }
 
-    /**
-     * @deprecated Will be removed in version 2.x. Use {@link CaseDetails#dataClassification} instead.
-     */
-    @Deprecated
-    @JsonGetter("security_classifications")
-    @ApiModelProperty("Deprecated. Use `data_classification` instead.")
-    public Map<String, JsonNode> getSecurityClassifications() {
-        return dataClassification;
-    }
-
+    @JsonIgnore
```

Comment:
We **don't** want to ignore this - we want only the `data_classification`
```suggestion
```

Reply:
Done